### PR TITLE
List of shopping lists endpoint

### PR DESCRIPTION
**Pull request by mronauli**
## Who worked on this

- [x] Alexis
- [x] Maria
- [ ] Ed
- [ ] Cody

## Type of change

- [ ] Style (Non-breaking Style code)
- [ ] Testing (Non-breaking Testing code)
- [x] Bug fix or refactor (change which fixes an issue)
- [ ] New feature (change which adds functionality)

## Summary of the change.

🔥 modify the item search method to pull values with medium, front facing urls

Note any questions or struggles as well.

## Checklist:

- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code if necessary

### CODE REVIEW

File: trackbasket_be/models/krogerservice.py [hunk 1]
```diff
@@ -69,16 +69,14 @@ def item_search(cls, term, at_risk_user_id):
       data['name'] = item['description']
       data['productId'] = item['productId']
       data['upc'] = item['upc']
-      
-      for price in item['items']:
-        data['unit_price'] = price['price']['regular']
-        
-      for aisle in item['aisleLocations']:
-        data['aisle_number'] = int(aisle['number'])
```
Comment (mronauli) [inline trackbasket_be/models/krogerservice.py:77]: remove unnecessary for loops, no need to iterate further. just grab first element in array

File: trackbasket_be/models/krogerservice.py [hunk 2]
```diff
@@ -69,16 +69,14 @@ def item_search(cls, term, at_risk_user_id):
...
-        
+      data['aisle_number'] = int(item['aisleLocations'][0]['number']) 
+      data['unit_price'] = item['items'][0]['price']['regular']
       for image in item['images']:
-        data['image'] = image['sizes'][0]['url']    
-      
+        if image['perspective'] == 'front': 
+          for size in image['sizes']: 
+            if size['size'] == 'medium':
+              data['image'] = size['url']
```
Comment (mronauli) [inline trackbasket_be/models/krogerservice.py:78]: specify value needed from hash to return the correct url

File: trackbasket_be/models/krogerservice.py (hunk 1 above)
Comment (adumortier) [reply trackbasket_be/models/krogerservice.py:77]: looks nice and concise

File: trackbasket_be/models/krogerservice.py (hunk 2 above)
Comment (adumortier) [reply trackbasket_be/models/krogerservice.py:78]: I think we might be able to convert the part of the response to a string and split that string to extract the information we need without having a nested loop

**Pull request merged**